server/admin-function/add-user.ts

```diff
@@ -0,0 +1,32 @@
+"use server";
+
+import { dbAdmin } from "@/lib/firebase-admin";
+
+export const addUser = async (domain: string, email:string, userId:string, name:string, imageURL:string): Promise<boolean> => {
+  try {
+    const collectionRef = dbAdmin.collection(`domain/${domain}/users`);
+    console.log('Add-user:', domain, userId);
+    if (await collectionRef.doc(userId).get().then(doc => doc.exists)){
+      return false;
```

Comment:
why return false instead of return the user data? You're already fetching it so you might as well get the data and return it

Reply:
it shouldnt be there, but the addUser returns a boleean to indicate if the addUser worked, when the addUser is called, we already have the information on the user, we needed it to add it